### ISSUE TITLE
Issue file not found when running ts_test/calibration

### ISSUE THREAD

**Issue by kxp** [saper-2/rpi-5inch-hdmi-touchscreen-driver#4]
I am trying to get everything working in the raspbian(latest and fully updated) in RPI 3, with the 5'' touchscreen, and almost everything works.
My issues happens when I try to run `sudo TSLIB_CONFFILE=/etc/ts.conf TSLIB_FBDEVICE=/dev/fb0 ts_test` or `sudo TSLIB_CONFFILE=/etc/ts.conf TSLIB_CALIBFILE=/etc/pointercal TSLIB_FBDEVICE=/dev/fb0 TSLIB_TSDEVICE=/dev/input/event2 ts_calibrate` I got the following error:
> fopen: No such file or directory
> Couldn't load module linear
> ts_config: No such file or directory

The touch works but I can't calibrate it, do you have any idea what can be?

**Comment by saper-2** [comment]
Now I missed this really bad.... I am sorry. 
Do you have files on specified paths? What tslib version are you using (from where?) - have you installed complet tslib, for me this smell like you used a raspbian tslib and didn't installed it fully or the path are different than from compiled by myself tslib. 
Also, take note, that I didn't tested this on latest raspbian images.

**Comment by kxp** [comment]
That's ok :)
I followed all of the steps using the same paths that were provided in the example, but at some point I gave up. 
Its important to say that I was using the most recent version of Raspbian.It can be related because the drivers that the manufacturer provide also doesn't work with the newest version of Raspbian.
In any case I end up using an older version of Raspbian with the driver that the manufacturer provided.
